python/triton/compiler/code_generator.py

```diff
@@ -1070,7 +1070,9 @@ def str_to_ty(name):
         return language.pointer_type(ty)
     tys = {
         "fp8e4nv": language.float8e4nv,
+        "fp8e4b8": language.float8e4b8,
         "fp8e5": language.float8e5,
+        "fp8e5b16": language.float8e5b16,
         "fp8e4b15": language.float8e4b15,
         "fp8e4b15x4": language.float8e4b15x4,
```

Comment:
We might want to remove float8e4b15x4 as now we clean up the matching between triton fp8 types and MLIR fp8 types and there is no corresponding MLIR fp8 types for it.